modules/virt-exposing-pci-device-in-cluster-cli.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-exposing-pci-device-in-cluster-cli_{context}"]
+= Exposing PCI host devices in the cluster using the CLI
+
+To enable the availability of PCI host devices in the cluster, add details about the PCI devices to the `HyperConverged` custom resource (CR). The PCI devices are exposed and permitted to be used in the cluster.
```

Comment:
"To enable the availability of" strikes me as a bit wordy, and this paragraph seems like it could be shortened to just one sentence. I wonder if something like this could work (completely optional):

"To expose PCI devices to the cluster so that they can be used, add them to the `spec.permittedHostDevices` stanza of the `HyperConverged` custom resource (CR)."